CMakeLists.txt

```diff
@@ -122,6 +126,8 @@ endforeach()
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
+# Static variants as PIC
```

Comment:
If you want this in your build you should just set this in your cmake command line. I don't see a good reason to hardcode it for all users.

Reply:
* Removed.